azure/azure-explanation/security-overview.rst

```diff
@@ -0,0 +1,109 @@
+of each image can be further enhanced with `Ubuntu Pro`_.
+
+Server
+~~~~~~
+
+Optimised for general server use cases, Ubuntu Server is ideal for users seeking a stable and secure OS foundation to
+support server workloads.
+
+Minimal
+~~~~~~~
+
+By providing a greatly reduced package set to minimise the security cross-section of the image, Ubuntu Minimal requires
+fewer security updates over time.
+
+CIS
+~~~
+
+A hardened Ubuntu image (see `Image Hardening`_), pre-configured to align with industry best practices and guidelines
+set by the Center for Internet Security (CIS). Ubuntu CIS images on Azure use custom tailoring files to meet both CIS
+benchmarks as well as ensure the image remains functional in Azure cloud environments.
+
+FIPS
+~~~~
+
+The Ubuntu FIPS image contains a Ubuntu kernel and security components which have been certified by the National
+Institute of Standards and Technology (NIST), making it a compelling choice for users operating in regulated
+environments such as FedRAMP, HIPAA, or PCI.
+
+Confidential VM
+~~~~~~~~~~~~~~~
+
+Supports Azure Confidential Virtual Machines, providing enhanced security features to protect data at rest, in use, and
+during boot.
+
+Ubuntu Pro
+----------
+
+Ubuntu Pro extends the capabilities of Ubuntu Server by incorporating advanced security, compliance features, and
+systems management tooling. It is designed for production environments where additional security layers and
+compliance with regulatory standards are paramount. Ubuntu Pro includes:
+
+* Expanded Security Maintenance: Security updates and patches for a wide array of applications and packages beyond the
+  standard offerings.
+* Kernel Livepatch: Minimises downtime and disruption by providing kernel updates that can be applied without rebooting
+  the system.
+* Up to 12 years of support: Long-term support that ensures your systems are secure and stable for more than a decade,
+  with a guaranteed upgrade path.
+
+To learn more about how to get Ubuntu Pro, see :doc:`../azure-how-to/instances/get-ubuntu-pro`.
+
+Image Hardening
+---------------
+
+The security profile for Ubuntu images on Azure can be further enhanced with `hardening <https://ubuntu.com/blog/what-is-system-hardening-definition-and-best-practices>`_
+by applying CIS hardening profiles from the `Ubuntu Security Guide (USG) <https://ubuntu.com/security/certifications/docs/usg>`_.
+
+Updates
+-------
+
+Latest Images
+~~~~~~~~~~~~~
+
+Ubuntu images for a given Azure Marketplace offer (see :doc:`../azure-how-to/instances/find-ubuntu-images`) are
+regularly published to deliver:
+
+* The latest kernel available for the image type.
+* Security patches to address CVEs.
+* The latest stable package set from the Ubuntu archives.
+
+Existing Workloads
+~~~~~~~~~~~~~~~~~~
+
+Deployments based off older image versions can receive the same security updates as the latest images by:
+
+* Keeping installed packages up to date with :code:`apt`.
```

Comment:
Do we have a doc page explaining the basics of `apt` somewhere. If so please add a link to it. If not, I would go further and tell people to run `sudo apt update` and `sudo apt [full-]upgrade -y` and briefly (short comment) describe what they do.
It might not seem the right place to do that but keeping systems up to date is the very first security principle that everyone should apply, so it's not wrong to clarify things here.

Reply:
I would end up re-linking to the Introduction to Security on Ubuntu document, which walks through all of these details. I would prefer to avoid re-iterating Ubuntu fundamentals in an Azure Security Overview document. This ultimately goes back to "what is the purpose of this document".